src/pages/Home.tsx

```diff
@@ -110,13 +113,14 @@ export default () => {
               align-items: center;
             `}
           >
-            <Typography variant="sectionHeader" bold color="primary">
+            <Typography variant="sectionHeader" bold color="primary" css={null}>
               Workflow runs
             </Typography>
           </div>
           <RunsTable runs={data?.runs || []} setLoading={setLoading} />
         </Container>
         <Container
+          css={null}
```

Comment:
just curious bout all these `css={null}`, why are they needed?

Reply:
These were weird. When I upgraded uikit to 1.7.0 I got a bunch of TypeScript compile errors complaining that any of the `Typography` and `Container` instances were missing `css`. Which was very strange, since it's not required when I look at the component itself inside platform/ui-kit :\

Any idea what's going on there?